qubes/storage/lvm.py

```diff
@@ -339,6 +339,12 @@ async def _reset(self):
         assert not self.snap_on_start and not self.save_on_stop, \
             "Not a volatile volume"
         self.log.debug('Resetting volatile %s', self.vid)
+        if self.encrypted_name is not None:
+            # self.encrypted_volume_path doesn’t actually use either argument,
+            # so just pass empty strings 🙂
+            await qubes.utils.cryptsetup(
+                    '--', 'close', self.encrypted_volume_path('', ''))
+            self.encrypted_name = None
```

Comment:
This isn't needed. `_reset()` is called only from `start()`, and with `start_encrypted()` already has this very call before calling `start()`.
And indeed tests complain about failing (the second) cryptsetup call.